docs/migrating-v1beta1-to-v1.md

```diff
@@ -0,0 +1,80 @@
+<!--
+---
+linkTitle: "Migrating from Tekton v1beta1"
+weight: 4000
+---
+-->
+
+# Migrating From Tekton `v1beta1` to Tekton `v1`
+
+- [Changes to fields](#changes-to-fields)
+- [Replacing `PipelineRunCancelled` with `Cancelled`](#replacing-pipelineruncancelled-status-with-cancelled)
+- [Upgrading `PipelineRun.Timeout` to `PipelineRun.Timeouts`](#upgrading-pipelinerun.timeout-to-pipelinerun.timeouts)
+- [Replacing `Pipeline.Tasks[].Conditions` with `WhenExpressions`](#replacing-pipeline.tasks[].conditions-with-whenexpressions)
+- [Deprecating Resources from Task, TaskRun, Pipeline and PipelineRun](#deprecating-resources-from-task,-taskrun,-pipeline-and-pipelinerun)
+- [Replacing `taskRef.bundle` and `pipelineRef.bundle` with Bundle Resolver](#replacing-taskRef.bundle-and-pipelineRef.bundle-with-bundle-resolver)
+
+
+This document describes the differences between `v1beta1` Tekton entities and their
+`v1` counterparts. It also describes the changed fields and the deprecated fields into v1.
+## Changes to fields
+
+In Tekton `v1`, the following fields have been changed:
+
+| Old field | New field |
+| --------- | ----------|
+| `spec.taskRunSpecs.taskServiceAccountName` | `spec.taskRunSpecs.serviceAccountName` |
+| `spec.taskRunSpecs.taskPodTemplate` | `spec.taskRunSpecs.podTemplate` |
+| `status.taskResults` | `status.results` |
+| `status.pipelineResults` | `status.results` |
+| `taskRun.spec.taskRef.bundle` | removed from `TaskRun` |
+| `pipelineRun.spec.pipelineRef.bundle` | removed from `PipelineRun` |
+
+## Replacing `PipelineRunCancelled` status with `Cancelled`
+`Cancelled` status should be used for replacing the deprecated `PipelineRunCancelled`.
+
+## Upgrading `PipelineRun.Timeout` to `PipelineRun.Timeouts`
+If you are using `pipelineRun.spec.timeout` you must now use `pipelineRun.spec.timeouts` as `pipelineRun.spec.timeout` is deprecated. 
+
+## Replacing `Pipeline.Tasks[].Conditions` with `WhenExpressions`
+You must use `WhenExpressions` if you are using `Pipeline.Tasks[].Conditions`
+
+```yaml
+# Pipeline.yaml (v1)
+  pipelineSpec:
+    tasks:
+      - name: create-file
+        when:
+          - input: "$(params.path)"
+            operator: in
+            values: ["README.md"]
+```
+
+## Deprecating `resources` from Task, TaskRun, Pipeline and PipelineRun
+The `resources` fields of Task, TaskRun, Pipeline and PipelineRun should not be used due to the deprecation of `PipelineResources`.
+
+## Replacing `taskRef.bundle` and `pipelineRef.bundle` with Bundle Resolver
+Bundle resolver in remote resolution should be used instead of `taskRun.spec.taskRef.bundle` and `pipelineRun.spec.pipelineRef.bundle`
```

Comment:
This also requires enabling the `enable-bundles-resolver` feature flag; please link to https://github.com/tektoncd/pipeline/blob/main/docs/install.md#customizing-the-pipelines-controller-behavior

@abayer would you mind adding the new resolvers flags to that section?